pkg/repository/v1/sqlcv1/filters-overwrite.sql

```diff
@@ -0,0 +1,15 @@
+-- name: ListFilters :many
+WITH inputs AS (
+    SELECT
+        UNNEST(@tenantIds::UUID[]) AS tenant_id,
+        UNNEST(@workflowIds::UUID[]) AS workflow_id,
+        -- NOTE: this is nullable, so sqlc doesn't support casting to a type
+        UNNEST(@scopes::TEXT[]) AS scope
+)
+
+SELECT f.*
+FROM v1_filter f
+JOIN inputs i ON (f.tenant_id, f.workflow_id, f.scope) = (i.tenant_id, i.workflow_id, i.scope)
```

Comment:
this is important - if we get a `null` scope when we run this query, that scope will never match a filter. I think this is the desired behavior since we don't want to (e.g.) do the opposite and end up iterating through a zillion possible filters looking for the one that evals to `true`. and this also means that if someone does the "default" thing (pushing an event with no filters and no scopes), they get the behavior they expect (the task runs)